pkg/cmd/provisioning/alibabacloud/README.md

```diff
@@ -0,0 +1,83 @@
+# Alibaba Cloud Manual Mode
+
+This is the guide for using  manual mode on alibaba cloud, for more use info about manual mode, please reference to [cco-mode-manual](https://docs.openshift.com/container-platform/4.9/authentication/managing_cloud_provider_credentials/cco-mode-manual.html).
+
+In alibaba cloud manual mode,  the CCO utility (`ccoctl`) binary would generate long-lived RAM AK credentials for individual OpenShift Container Platform cluster components. The ram user who owns the AK would be attached the ram policy with the permission defined in each components, and a root ram user who required `ram:CreatePolicy` and `ram:AttachPolicyToUser` pemission as least is needed for attaching the permission for each component.
+
+## Prerequisite
+
+1. Extract and prepare the ccoctl binary from the release image.
+
+2. [create a ram user](https://partners-intl.aliyun.com/help/doc-detail/93720.htm)(has no permission in default) for binding the cluster components permission, and get the ram accesskey id/accesskey secret/user name as the input parameters of the alibaba cloud ccoctl command.
+
+3. choose an existing ram user who required `ram:CreatePolicy` and `ram:AttachPolicyToUser` pemission as least, and get this ram user's accesskey id/accesskey secret for attaching the specific component permission to the ram user created in step 2.
+
+    
+
+## Procedure
+
+1. Extract the list of CredentialsRequest custom resources (CRs) from the OpenShift Container Platform release image:
+
+   ```bash
+   $ oc adm release extract --credentials-requests --cloud=alibabacloud --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+   
+   ```
+
+2. For each CredentialsRequest CR in the release image, ensure that a namespace that matches the text in the spec.secretRef.namespace field exists in the cluster. This field is where the generated secrets that hold the credentials configuration are stored.
```

Comment:
Item 2 and 3 are really only necessary when preparing for an OpenShift upgrade to an existing cluster. When doing a fresh installation, the installer/bootstrap node/CVO will create any necessary Namespaces as part of the initial installation.
So you can mark these two steps as only being needed as part of pre-cluster-upgrade steps.